test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -202,7 +202,7 @@ eq_tends(
     pv::PV,
     m::EDMF,
     ::Flux{SecondOrder},
-) where {PV <: Union{Momentum, Energy, TotalMoisture}} = ()  # do _not_ add SGSFlux back to grid-mean
+) where {PV <: Union{Momentum, Energy, TotalMoisture}} = (SGSFlux{PV}(),)   # do _not_ add SGSFlux back to grid-mean
```

Comment:
Note to self: only for testing

Reply:
this entire branch is just for testing now